test_framework/terraform/harvester/sles/variables.tf

```
@@ -0,0 +1,54 @@
+variable "lab_url" {
+  type        = string
+  description = "LAB URL"
+  sensitive   = true
+}
+
+variable "lab_access_key" {
+  type        = string
+  description = "LAB ACCESS_KEY"
+  sensitive   = true
+}
+
+variable "lab_secret_key" {
+  type        = string
+  description = "LAB SECRET_KEY"
+  sensitive   = true
+}
+
+variable "ssh_public_key_file_path" {
+  type        = string
+  default     = "~/.ssh/id_rsa.pub"
+}
+
+variable "arch" {
+  type        = string
+  description = "available values (amd64, arm64)"
+  default     = "amd64"
+}
+
+variable "os_distro_version" {
+  type        = string
+  default     = "15-sp6"
+}
+
+variable "k8s_distro_name" {
+  type        = string
+  default     = "rke2"
+  description = "kubernetes distro version to install [rke, k3s]  (default: rke2)"
```

`rke` should be a typo.
```suggestion
  description = "kubernetes distro version to install [rke2, k3s]  (default: rke2)"
```